src/cmd-buildextend-live

```diff
@@ -422,6 +418,9 @@ def generate_iso():
         # can be EFI/fedora or EFI/redhat
         grubpath = ensure_glob(os.path.join(tmpisoroot, 'EFI/*/grub.cfg'))
```

Comment:
Let's add an `assert len(grubpath) == 1` here? That makes it easier to reason about the following old and new code.